github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
```

Comment:
That's a matter of taste. I like this one more. The guides that inspired our guides also use this syntax.